src/Cases/index.js

```diff
@@ -114,7 +115,20 @@ export default () => {
         config.defaults.country
       );
     }
-  }, [activeTab, dataLoaded]);
+  }, [
+    activeTab,
+    caseDates,
+    config.defaults.countries,
+    config.defaults.country,
+    dataLoaded,
+    dates,
+    selectedCountryId,
+    selectedDateIndex,
+    setCountrySelectEntries,
+    setDates,
+    setSelectedCountryId,
+    setSelectedDateIndex,
+  ]);
```

Comment:
Is this change coming from some auto-formatting? I saw this in another PR - this breaks the page, the only thing fire this useEffect should be the dataLoaded and activeTab changes.